packages/adk-core/test/core.test.ts

```diff
@@ -59,6 +59,16 @@ describe('@aws/codecatalyst-adk-core', () => {
         expect(process.exitCode === 1).toBeTruthy();
     });
 
+    it('test command with pipe', () => {
+        const output = adkCore.command('echo "Hello" | grep "lo"');
```

Comment:
what happens if the command is just?
```
echo 'this is some string | . hello`
```

Reply:
I don't quite understand this command here, are you just trying to echo a string with `|` in it? It doesn't matter, it will just run the echo command, print out `this is some string | . hello`

Reply:
`are you just trying to echo a string with | in it`. yup.

we are checking if the command contains a pipe. if it does, disable the disableStdInput. However, the command that i used as an example, is just a simple string and it does not contain any pipe.

but the current logic will still disable it because the string contains ` | `.

Reply:
Discussed offline:
```
No it doesn't matter, the disableStdInput won't interrupt the command execution, it just make the command execution cleaner, if we have | in the command, the command will be executed normally except that the output is not cleaned up but this shouldn't be an issue

I have tried the case where we echo a line with | in it to verify, the echo command works normally.
```